### PR TITLE
Order page targeting properties to make it easier to compare to DCR

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Order page targeting properties to make it easier to compare to DCR

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js [hunk 1]
```diff
@@ -264,41 +264,41 @@ const rebuildPageTargeting = () => {
...
-            // Indicates whether the page is DCR eligible. This happens when the page
-            // was DCR eligible and was actually rendered by DCR or
-            // was DCR eligible but rendered by frontend for a user not in the DotcomRendering experiment
+            fr: getVisitedValue(),
+            gdncrm: getUserSegments(adConsentState),
             inskin: inskinTargetting(),
+            ms: formatTarget(page.source),
+            permutive: getPermutiveSegments(),
+            pv: config.get('ophan.pageViewId'),
+            // round video duration up to nearest 30 multiple
```
Comment (zekehuntergreen) [inline static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js:288]: this comment applies to the `vl` key, right?

Comment (arelra) [reply static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js:288]: Good 👀

**Review by ashishpuliyel — DISMISSED** [review]
~~The comment             // round video duration up to nearest 30 multiple
is no longer near the actual param (vl). Other than that it looks fine :)~~
Zeke already spotted this above

File: static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js (hunk 1 above)
Comment (ashishpuliyel) [reply static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js:288]: Booo. You beat me to it!

**Comment by arelra** [comment]
> ~The comment // round video duration up to nearest 30 multiple is no longer near the actual param (vl). Other than that it looks fine :)~
> Zeke already spotted this above

Fixed and updated the other comments to be a bit more descriptive

**Review by zekehuntergreen — APPROVED** [review]

**Pull request merged**